product-service/src/functions/post-products/handler.ts

```diff
@@ -0,0 +1,85 @@
+import 'source-map-support/register';
+import {formatJSONResponse, ValidatedEventAPIGatewayProxyEvent} from '@libs/apiGateway';
+import {middyfy} from '@libs/lambda';
+import {Pool} from 'pg';
+import schema from '@functions/post-products/schema';
+
+const {PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD} = process.env;
+const dbOptions = {
+    host: PG_HOST,
+    port: PG_PORT,
+    database: PG_DATABASE,
+    user: PG_USERNAME,
+    password: PG_PASSWORD,
+    ssl: {
+        rejectUnauthorized: false
+    },
+    connectionTimeoutMillis: 1000
+};
+
+const postProducts: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+    console.log(event);
+    let newProductId = null;
+    try {
+        const productData = event.body;
+        const pool = new Pool(dbOptions);
+        if (!productData || !productData.title || !productData.price || !productData.count) {
```

Comment:
You have already validated your request body by the schema (sting 20). I think that is not necessary check.